navigation3/navigation3-ui/src/nonAndroidMain/kotlin/androidx/navigation3/ui/LocalNavigationEventDispatcherOwner.nonAndroid.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2025 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.navigation3.ui
+
+import androidx.compose.runtime.Composable
+import androidx.navigationevent.NavigationEventDispatcherOwner
+
+@Composable
+internal actual fun findViewTreeNavigationEventDispatcherOwner(): NavigationEventDispatcherOwner? {
+    // TODO("Not yet implemented")
+    return null
```

Comment:
Do we need to keep `null` here? Is that TODO needs to be addressed? If so please add YT task for that